src/Illuminate/Validation/Validator.php

```diff
@@ -385,17 +385,21 @@ protected function validateAttribute($attribute, $rule)
         // If we have made it this far we will make sure the attribute is validatable and if it is
         // we will call the validation method with the attribute. If a method returns false the
         // attribute is invalid and we will add a failure message for this failing attribute.
-        $validatable = $this->isValidatable($rule, $attribute, $value);
+        if (! $this->isValidatable($rule, $attribute, $value)) {
+            return;
+        }
+
+        if (is_a($rule, RuleContract::class, true)) {
+            if (is_string($rule)) {
+                $rule = app($rule, $this->parseNamedParameters($parameters));
+            }
 
-        if ($rule instanceof RuleContract) {
-            return $validatable
-                    ? $this->validateUsingCustomRule($attribute, $value, $rule)
-                    : null;
+            return $this->validateUsingCustomRule($attribute, $value, $rule);
         }
 
         $method = "validate{$rule}";
 
-        if ($validatable && ! $this->$method($attribute, $value, $parameters, $this)) {
+        if (! $this->$method($attribute, $value, $parameters, $this)) {
```

Comment:
```suggestion
        if (!$this->$method($attribute, $value, $parameters, $this)) {
```

Reply:
This would cause CI failures.

Reply:
This isn't the way this is styled throughout the framework and StyleCI will fail if this is done.